public/robots.txt

```diff
@@ -3,3 +3,5 @@
 # To ban all spiders from the entire site uncomment the next two lines:
 # User-agent: *
 # Disallow: /
+
+Sitemap: http://library.ucsd.edu/hyrax/sitemap.xml
```

Comment:
Since as you rightly pointed out, this won't be used until we hit production, and there's no real "configuration" here to speak of, perhaps we should just put the ultimate production URL in place now? Unless there's a good reason not to, as far as testing, etc. goes. 

`Sitemap: http://library.ucsd.edu/dc/sitemap.xml`